webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -61,7 +61,7 @@ const isNonK8RMError = (e: unknown): boolean => {
 };
 
 const isNotAuthorizedErr = (e: unknown): boolean => {
-  return e instanceof DetError && e.sourceErr instanceof Response && e.sourceErr['status'] === 403;
+  return e instanceof DetError && e.sourceErr instanceof Response && (e.sourceErr['status'] === 403 || e.sourceErr['status'] === 401);
```

Comment:
**question:**
- is there enough overlap of this and `isAuthFailure`?
- not for this PR but maybe this function can be made reusable into `src/utils/services.ts` if this is deemed common enough of a situation?